libsoftwarecontainer/component-test/softwarecontainerlib_componenttest.cpp

```diff
@@ -44,18 +45,20 @@ class SoftwareContainerApp : public SoftwareContainerLibTest
 
 public:
 
-    void startGateways(const std::string &config, const std::string &gatewayID)
+    bool startGateways(const std::string &config, const std::string &gatewayID)
     {
         json_error_t error;
         json_t *configJson = json_loads(config.c_str(), 0, &error);
-        ASSERT_FALSE(configJson == nullptr);
+        assert(configJson != nullptr);
```

Comment:
doesn't ASSERT_NE working here ? As far as i remember we used to use that function instead of this